applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -506,14 +517,29 @@ int le_audio_user_defined_button_press(enum le_audio_user_defined_action action)
 	return ret;
 }
 
-int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate)
+int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate, uint32_t *pres_delay)
 {
 	if (active_stream.codec == NULL) {
-		return -ECANCELED;
+		LOG_WRN("No active stream to get config from");
+		return -ENXIO;
+	}
+
+	if (bitrate == NULL && sampling_rate == NULL && pres_delay == NULL) {
+		LOG_ERR("No valid pointers received");
+		return -ENXIO;
 	}
 
-	*sampling_rate = active_stream.codec->frequency;
-	*bitrate = active_stream.codec->bitrate;
+	if (sampling_rate != NULL) {
+		*sampling_rate = active_stream.codec->frequency;
+	}
+
+	if (bitrate != NULL) {
+		*bitrate = active_stream.codec->bitrate;
+	}
+
+	if (pres_delay != NULL && active_stream.stream != NULL) {
```

Comment:
This one is a little dangerous. If `pres_delay != NULL,` but `active_stream.stream == NULL` pres_delay will not be set and the function will return 0.

Reply:
I'll change it to
```suggestion
	if (pres_delay != NULL) {
		if (active_stream.stream == NULL) {
```